hoomd/md/methods/methods.py

```diff
@@ -27,10 +27,20 @@ class Method(AutotunedObject):
 
     def _attach_hook(self):
         self._simulation.state.update_group_dof()
+        self.validate_group()
 
     def _detach_hook(self):
         self._simulation.state.update_group_dof()
 
+    def validate_group(self):
+        """Verify groups.
+
+        Groups may change after attaching.
+        Users can call `validate_group` to verify the groups after changing
+        them.
+        """
```

Comment:
Change to clarify that only checks that constituent particles are not integrated.